src/Command/ToolsToWrite.php

```
@@ -0,0 +1,113 @@
+<?php
+/**
+ * Created by PhpStorm.
+ * @author: Samir Subašić
+ * Date: 13.10.2017
+ * Time: 1:06
+ */
+
+namespace BOF\Command;
+
+use Symfony\Component\Filesystem\Filesystem;
+use Symfony\Component\Filesystem\Exception\IOExceptionInterface;
+
+
+class ToolsToWrite
+{
+    private $reportList;
+
+    public static function writeToFile($firstRow, $reportList)
+    {
+        $fs = new Filesystem();
+
+        try {
+            $text = ToolsToWrite::prepairTable($firstRow, $reportList);
+            $fs->dumpFile('file.txt', $text);
+        } catch (IOExceptionInterface $e) {
+            echo "An error occurred while creating file.txt ".$e->getFile();
```

This is really not a good way to handle exceptions:
1. Using echo in an MVC framework is a big violation.
2. What if we don't want to display the error to the user?
3. There is no way for the caller of this method to know that the error has occurred.

It would be much better to just throw an exception and then let the caller of the class decide what to do with.